documentation/asciidoc/computers/camera/rpicam_options_common.adoc

```diff
@@ -44,533 +48,515 @@ Available cameras
                              4056x3040 [10.00 fps - (0, 0)/4056x3040 crop]
 ----
 
-In the above example, the IMX219 sensor is available at index 0 and IMX477 at index 1. The sensor mode identifier takes the following form:
+Sensor mode identifiers have the following form:
+
 ----
 S<Bayer order><Bit-depth>_<Optional packing> : <Resolution list>
 ----
-For the IMX219 in the above example, all modes have a `RGGB` Bayer ordering and provide either 8-bit or 10-bit CSI2 packed readout at the listed resolutions. The crop is specified as (<x>, <y>)/<Width>×<Height>, where (x, y) is the location of the crop window of size width × height in the sensor array. The units remain native sensor pixels, even if the sensor is being used in a binning or skipping mode.
 
-----
-	--camera			Selects which camera to use <index>
-----
+For the IMX219 in the above example:
 
-The `--camera` option will select which camera to use from the supplied <index> value. The <index> value can be obtained from the `--list-cameras` option.
+* all modes have an `RGGB` Bayer ordering
+* all modes provide either 8-bit or 10-bit CSI2 packed readout at the listed resolutions
+* crop, specified as `(<x>, <y>)/<Width>×<Height>`, where `(x, y)` is the location of the crop window of size width × height in the sensor array. The units remain native sensor pixels, even if the sensor is used in binning or skipping mode.
```

Comment:
I don't think this bullet-point is specific to IMX219, so perhaps it ought to be moved above "For the IMX219..." line?

Reply:
Excellent point, how did I overlook this?